### ISSUE TITLE
Cards being generated but not displayed in anki droid

### ISSUE THREAD

**Issue by Sigfr1ed** [infinyte7/Anki-Occlusion#13]
Hi,

I have installed anki image occlusion per the instructions but the only thing I get is a weird screen - https://drive.google.com/file/d/19aWbuDxBkog5wgjXUECtOCvSEF6E_kmh/view?usp=drive_link

I double checked that all the permissions were granted but its not working. Could you please help me find out why this is not working?

**Comment by krmanik** [comment]
> Write access to all internal storage directories⁠ except /Android/data/, /sdcard/Android, and most subdirectories of /sdcard/Android. This write access includes [direct file path](https://developer.android.com/training/data-storage/shared/media#direct-file-paths) access. 
https://developer.android.com/training/data-storage/manage-all-files

Due to scope storage the directory is not accessible to other apps. AnkiDroid now stores the collection.media file in `Android/data/com.ichi2.anki/files/AnkiDroid/collection.media`, so it is difficult to implement fix for this (also out of time to maintain this project).
Also, the image occlusion feature will be available in next version of AnkiDroid. For now, view solution in this issues https://github.com/krmanik/Anki-Occlusion/issues/12.

In summary you have to copy images with file name start with `cordova-*` from `AnkiDroid/collection.media` to `Android/data/com.ichi2.anki/files/AnkiDroid/collection.media`.
